backend/settings.py

```diff
@@ -122,11 +112,11 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.postgresql",
-        "NAME": os.environ.get("DATABASE_NAME", "portail"),
-        "USER": os.environ.get("DATABASE_USER", "root"),
-        "PASSWORD": os.environ.get("DATABASE_PASSWORD", "password"),
-        "HOST": os.environ.get("DATABASE_HOST", "localhost"),
-        "PORT": os.environ.get("DATABASE_PORT", "5432"),
+        "NAME": env.str("DATABASE_NAME"),
```

Comment:
Since we don't put default values, maybe create a `.env.dist` file with dummy values so we know what fields we need to fill

Reply:
Good idea!